sass/lightbox.scss

```diff
@@ -65,6 +65,11 @@ $button-padding: 40px;
 
   } // &.single
 
+  &.extras-hidden #lightbox-img-wrap > :not(img) {
```

Comment:
I think this selector needs to be as specific as it is here. I had is as `#lightbox-wrap.extras-hidden .control` at first, but there's a rule for `#lightbox-img-wrap .next:hover` which is more specific, and it overrides it. Also, this is a lightbox-wide toggle `.extras-hidden`, so it's applied on the top-level class. But the desire is to hide all top-level nodes except the image (with the soon-to-be-implemented addition of avatar will have more than just the controls there).